app/core/ingestion/processor.py

```diff
@@ -78,15 +98,26 @@ def ingest_event_row(
     return result
 
 
-def initialise_context(db: Session) -> IngestContext:
+def initialise_context(db: Session, org_name: str) -> IngestContext:
     """
     Initialise the database
 
     :return [IngestContext]: The organisation that will be used for the ingest.
     """
     with db.begin():
-        organisation = db.query(Organisation).filter_by(name="CCLW").one()
-        return IngestContext(org_id=cast(int, organisation.id), results=[])
+        organisation = db.query(Organisation).filter_by(name=org_name).one()
+        if org_name == "CCLW":
+            return CCLWIngestContext(
+                org_name=org_name, org_id=cast(int, organisation.id), results=[]
+            )
+        elif org_name == "UNFCCC":
+            return UNFCCCIngestContext(
+                org_name=org_name, org_id=cast(int, organisation.id), results=[]
+            )
+        else:
+            raise ValueError(
+                f"Code not in sync with data - org {org_name} unknown to code"
+            )
```

Comment:
Just a suggestion from the "pythonic" handbook :joy: 
```suggestion
        if org_name == "CCLW":
            return CCLWIngestContext(
                org_name=org_name, org_id=cast(int, organisation.id), results=[]
            )
        if org_name == "UNFCCC":
            return UNFCCCIngestContext(
                org_name=org_name, org_id=cast(int, organisation.id), results=[]
            )
        raise ValueError(
            f"Code not in sync with data - org {org_name} unknown to code"
        )
```

Reply:
Thanks - not sure what the advantage is - do you have a reference?